pymdownx/tasklist.py

```diff
@@ -75,17 +57,43 @@ def sub_paragraph(self, li):
             if first.tag == "p" and first.text is not None:
                 m = RE_CHECKBOX.match(first.text)
                 if m is not None:
-                    first.text = self.md.htmlStash.store(
-                        get_checkbox(m.group('state'), self.custom_checkbox, self.clickable_checkbox)
-                    ) + m.group('line')
+                    first.text = self.get_checkbox(m)
                     found = True
         return found
 
+    def get_checkbox(self, match):
+        """Get checkbox tag."""
+        state = match.group('state')
+
+        if self.custom_checkbox:
+            if self.clickable_label:
+                line = self.md.htmlStash.store(
+                    '<label><span class="task-list-control">' +
+                    '<input type="checkbox"{}{}/>'.format(
+                        ' disabled' if not self.clickable_checkbox else '',
+                        ' checked' if state.lower() == 'x' else '') +
+                    '<span class="task-list-indicator"></span></span>'
+                ) + match.group('line') + self.md.htmlStash.store("</label>")
```

Comment:
There, I break up the stash, so the matched group for the rest of the line gets properly rendered